### PR TITLE
MU-1557 timeOut attribute documentation

### PR DESCRIPTION
**Pull request by fernandothea**

### CODE REVIEW

File: modules/ROOT/pages/munit-test-concept.adoc [hunk 1]
```diff
@@ -96,6 +96,21 @@ MUnit also allows you to tag your tests, so you can choose to run tests under a
 .In Anypoint Studio, you can define tags in the proper field.
 image::munit-test-concept-c2d9f.png[test-tags]
 
+== About MUnit Test TimeOut Attribute
+
+MUnit tests will automatically fail if they take more than 2 minutes to complete. Sometimes, that time can be too long or too short. +
+In that case, you can use the timeOut attribute, where you can set the desired maximum time -in milliseconds- that the test can run before fail. For example:
```
Comment (diegostrubolini) [inline modules/ROOT/pages/munit-test-concept.adoc]: fail -> failing

Comment (diegostrubolini) [inline modules/ROOT/pages/munit-test-concept.adoc]: timeOut -> `timeOut`

File: modules/ROOT/pages/munit-test-concept.adoc [hunk 2]
```diff
@@ -96,6 +96,21 @@ MUnit also allows you to tag your tests, so you can choose to run tests under a
 .In Anypoint Studio, you can define tags in the proper field.
 image::munit-test-concept-c2d9f.png[test-tags]
 
+== About MUnit Test TimeOut Attribute
+
+MUnit tests will automatically fail if they take more than 2 minutes to complete. Sometimes, that time can be too long or too short. +
```
Comment (diegostrubolini) [inline modules/ROOT/pages/munit-test-concept.adoc]: Should we make clear that the timeOut is for each test, and not the whole run of all tests?

**Review by diegostrubolini — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/munit-test-concept.adoc (hunk 2 above)
Comment (fernandothea) [reply modules/ROOT/pages/munit-test-concept.adoc]: Hmm maybe we should talk about `test` (in singular) and not `tests`? Or explicitly say that?

Comment (diegostrubolini) [reply modules/ROOT/pages/munit-test-concept.adoc]: I would test about a single test

**Review by diegostrubolini — APPROVED** [review]

**Pull request merged**